superdesk/io/feed_parsers/ninjs.py

```diff
@@ -159,8 +159,9 @@ def parse_renditions(self, renditions):
             # Parse width and height
             width = rendition_data.get("width")
             height = rendition_data.get("height")
-            if isinstance(width, int) and isinstance(height, int):
```

Comment:
I think this might be better, either get both or none, otherwise later in the code we would have to check always if we have both

Reply:
got it